aslprep/niworkflows/viz/utils.py

```diff
@@ -403,8 +403,8 @@ def _compose_view(bg_svgs, fg_svgs, ref=0):
     else:
         newroots = roots
     fig.append(newroots)
-    fig.root.attrib.pop("width")
-    fig.root.attrib.pop("height")
+    fig.root.attrib.pop("width", None)
```

Comment:
does pop not automatically return None?

Reply:
It started breaking when I unpinned the svgutils version. I haven't checked what the `attrib` attribute is, so it might not be a regular dictionary. I copied the fix from the current version of niworkflows.